components/EmberPage.vue

```diff
@@ -176,6 +176,16 @@ export default {
       }
     },
 
+    getBooleanAttr(el, attr) {
+      if (!el) {
+        return false;
+      }
+
+      const v = el.getAttribute(attr);
+
+      return (v !== null) && (v !== 'false');
```

Comment:
Should this also handle other falsies? Apparently some old DOM spec's can return empty string for missing

Reply:
Switched back to how it was but with explicit check for true